bin/util.py

```diff
@@ -1048,17 +1031,21 @@ def exec_command(
     if 'timeout' in kwargs:
         if kwargs['timeout'] is None:
             timeout = None
-        elif kwargs['timeout'] is True:
-            # Use the default timeout.
-            pass
         elif kwargs['timeout']:
             timeout = kwargs['timeout']
         kwargs.pop('timeout')
     if timeout is not None and math.isinf(timeout):
         timeout = None
 
-    if (is_windows() or is_wsl()) and 'memory' in kwargs:
+    memory: Optional[int] = None
+    if 'memory' in kwargs:
+        if kwargs['memory'] is not None:
+            memory = kwargs['memory']
         kwargs.pop('memory')
+    if config.args.memory:
+        memory_limit = config.args.memory
```

Comment:
`memory`?

Reply:
I think it's already `memory` in `master`?